samples/orbit-2-posts/orbit-2-posts_build.gradle.kts

```diff
@@ -51,18 +49,8 @@ dependencies {
     implementation(project(":orbit-2-livedata"))
     implementation(project(":orbit-2-viewmodel"))
 
-    // Kotlin
-    implementation(ProjectDependencies.kotlinCoroutines)
```

Comment:
seems surprising we can remove this.

PS are we building the sample apps on CI at all? Maybe we should check

Reply:
I guess we are `./gradlew assemble`